gatsby-config.js

```diff
@@ -2,18 +2,19 @@ module.exports = {
   siteMetadata: {
     title: `Wayfair Open Source`,
     description: `Home of the Open Source Program Office and Wayfair specific documenting for working with OSS`,
-    author: ``,
-    siteUrl: `http://wayfair.github.io`,
+    author: `opensource@wayfair.com`,
+    siteUrl: `https://wayfair.github.io`,
+    keywords: ["opensource", "wayfair", "Wayfair", "OSS"],
   },
   plugins: [
     `gatsby-plugin-sitemap`,
     {
-      resolve: 'gatsby-plugin-robots-txt',
+      resolve: "gatsby-plugin-robots-txt",
```

Comment:
Non-blocking Nit: Seems like we are switching from Single Quotes to Double quotes here, while the majority of the rest of the config uses back ticks? any reason not to consistently use back ticks?

Reply:
nope just what I did by habit